packages/labs/router/src/lib/url-pattern.d.ts

```diff
@@ -0,0 +1,43 @@
+declare class URLPattern {
```

Comment:
```suggestion
/**
 * @license
 * Copyright 2021 Google LLC
 * SPDX-License-Identifier: BSD-3-Clause
 */
 
declare class URLPattern {
```

(Or if this is copied from somewhere else, add its copyright header)

Reply:
```suggestion
// See https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1199 to track adding URLPattern types to the standard TypeScript DOM types.

declare class URLPattern {
```

Reply:
This file will likely go away by the time the PR is done, by including the URLPattern polyfill.